app/controllers/manage/configs_controller.rb

```diff
@@ -17,7 +17,9 @@ def update
     value = params[:hackathon_config][key]
     value = true if value == "true"
     value = false if value == "false"
-    if @config.value != value
+    if @config.var.end_with?("_asset") && !value.start_with?('http://', 'https://')
+      redirect_to manage_configs_path, notice: "Config \"#{key}\" was not changed"
```

Comment:
```suggestion
      flash[:alert] = "Config \"#{key}\" must start with http:// or https://"
      render :edit
```

This should help it be more clear why the value was rejected, and using `render :edit` will render the same page (hopefully) with the same value the user tried to use